pset-iterators.js

```diff
@@ -114,7 +131,12 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
+const presidentsBetween1850and1900 = president18501900.filter(function(president){
+  return president.took_office
+  return president.left_office
+})
 
+console.log(presidentsBetween1850and1900)
```

Comment:
@ChicoMangusta Good attempt at this solution. One suggestion because you are using the filter and the variable `president18501900 ` this will return an error since `president18501900 ` variable is undefined.
You can replace `president18501900 ` with `presidents`

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)